packages/x-components/src/x-modules/popular-searches/wiring.ts

```diff
@@ -60,5 +61,5 @@ export const popularSearchesWiring = createWiring({
   },
   ExtraParamsChanged: {
     setPopularSearchesExtraParams
-  }
+  },
```

Comment:
As in the related tags module, we need to react to the `UserAcceptedAQueryPreview` event and update the extraParams in this module.

```suggestion
  },
  UserAcceptedAQueryPreview: {
    setPopularSearchesParamsFromQueryPreview
  }
```
and add the commit with extraParams as payload accordingly:
`export const setPopularSearchesParamsFromQueryPreview = wireCommit(
  'setParams',
  ({ eventPayload: { extraParams } }) => extraParams
);`